### ISSUE TITLE
On screen statistics: Issues with vertex count and suggested solutions

### ISSUE THREAD

**Issue by Pilvinen** [starandserpent/Voxel-demo#68]
We are currently showing in on screen statistics a vertex count for in-view vertices.

It seems to be working fine and AFAIK the number is pulled straight from Godot (?) so it should be efficient.

However there are so many vertices that the numbers turn into exponents and the numbers lose their meaning.

Displaying either **Polygons**, **Triangles** or **both** would make a lot more sense to me because the numbers would be more inherently smaller. They would also be easier to compare with GPU performance specs.

Personally I would opt for **both**.

Implementation needs to be **fast**. There cannot be any overhead.

Include both:

- In-view
- Loaded world.

**Comment by Pawlost** [comment]
Yes we are taking values from godot. But vertice count is only value which can be taken. This means it would require our own implementation.

**Comment by Pawlost** [comment]
Showing something in view is extremely hard right now so I am removing good first issue tag.
